rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -710,7 +709,7 @@ def refresh_activated_validators_deposits(self) -> None:
                 history_changes,
             )
             write_cursor.executemany(
-                'UPDATE evm_events_info SET extra_data=null WHERE identifier=?',
+                'UPDATE history_events SET extra_data=null WHERE identifier=?',
```

Comment:
You should change this. As you can see in this function `history_events` are already edited at line 707. Change the logic of this function to put the extra data change in `history_changes` and do only a single sqlite query to change the table. Not two.